src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezimageasset.html.twig

```diff
@@ -84,7 +84,7 @@
             locationId: destination_content.contentInfo.mainLocationId
         }) %}
 
-        {% set image_uri = ez_field_value(destination_content, ez_image_asset_content_field_identifier()).uri %}
+        {% set image_uri = ez_field_value(destination_content, ez_content_field_identifier_image_asset()).uri %}
```

Comment:
As I said the old name fits me better. In other names, we have the object first, then "value". If nobody shares my opinion, than I am ok with this

Reply:
You have also said that if we have more than one function in `ez_content_field_identifie` domain then you are ok with it. I decide to keep `ez_content_field_identifier_first_filed_image ` so we have two.